ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2226,6 +2236,12 @@ router_bgp:
       password: "< encrypted_password >"
       send_community: < standard | extended | large | all >
       maximum_routes: < integer >
+      maximum_routes_warning_limit: < integer >
+      maximum_routes_warning_limit_percent: < true | false >
+      maximum_routes_warning_only: < true | false >
+      allowas_in:
+          enabled: < true | false >
+          allow_local_asn_n_times: < 1-10 >
```

Comment:
Can you make this ```allowed_local_asns```

Reply:
We already had many goes at this key name. I'm not sure your's is more precise than the one in the PR. I also tried a few in the issue discussion, but never found a good one.

Reply:
It's tricky, if the goal is to make the key shorter, we could to something like

```
allowas_in:
  times: < 1-10 >
```

reads like: 

```
allow(local)as in < 1-10> times
```

Somehow I don't like to have "local asn**s**"(with plural s), because there is only one local ASN from the logical point of view.. it's just repeated n-times.

What do you think?

Reply:
Yeah, looks good to me. Either ```times``` or may be ```repeat``` works for me.

Reply:
changed to:

```
      allowas_in:
          enabled: < true | false >
          times: < 1-10 >
```